src/transformers/modeling_tf_albert.py

```diff
@@ -324,16 +330,23 @@ def __init__(self, config, **kwargs):
         )
         self.dropout = tf.keras.layers.Dropout(config.hidden_dropout_prob)
 
+        self.pre_layer_norm = config.pre_layer_norm
+
     def call(self, inputs, training=False):
         hidden_states, attention_mask, head_mask = inputs
 
+        if self.pre_layer_norm:
```

Comment:
I think this line (pre_layer_norm) should be moved after the call of self.attention (Line 340). There is already a pre_layer_norm process in TFAlbertAttention. According to the paper (ON LAYER NORMALIZATION IN THE TRANSFORMER ARCHITECTURE), there should be another pre_layer_norm before FFN sub-layer. Therefore I assume Line 338 should be followed by Line 340.